package.json

```diff
@@ -24,8 +27,5 @@
     "name": "JSTransformers Team",
     "url": "http://github.com/orgs/jstransformers/people"
   },
-  "license": "MIT",
-  "dependencies": {
-    "octet": "^3.0.0"
-  }
+  "license": "MIT"
```

Comment:
Was thinking, having this in place makes the diff smaller when updating Boilerplate in the future, since the `,` isn't there anymore.